frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```
@@ -317,6 +317,7 @@ string.GitMachete.RediscoverSuggester.backgroundable-check-task.title=Checking f
 
 # Machete file editor
 
+action.GitMachete.MacheteAnnotator.IntentionAction.create-nonexistent-branch=Create branch {0} from {1}
```

```suggestion
action.GitMachete.MacheteAnnotator.IntentionAction.create-nonexistent-branch=Create branch ''{0}'' from ''{1}''
```

AFAICS intentions always quote branch names etc.